Sources/JSONWebToken/Data+hash.swift

```diff
@@ -11,7 +11,7 @@ import CryptoKit
 
 extension Data {
     func sha(_ hashFunction: SignatureAlgorithm.HashFunction) -> Data {
-        if #available(iOSApplicationExtension 13.0, *) {
```

Comment:
this is a fairly significant change as it means this can no longer be used in an app extension (such as the share extension).  Is that really necessary?